applications/luci-app-tailscale/htdocs/luci-static/resources/view/tailscale/interface.js

```diff
@@ -0,0 +1,116 @@
+/* SPDX-License-Identifier: GPL-3.0-only
+ *
+ * Copyright (C) 2022 ImmortalWrt.org
+ * Copyright (C) 2024 asvow
+ */
+
+'use strict';
+'require dom';
+'require fs';
+'require poll';
+'require ui';
+'require view';
+
+return view.extend({
+	async load() {
+		const res = await fs.exec('/sbin/ip', ['-s', '-j', 'ad']);
+		if (res.code !== 0 || !res.stdout || res.stdout.trim() === '') {
+			ui.addNotification(null, E('p', {}, _('Unable to get interface info: %s.').format(res.message)));
+			return [];
+		}
+
+		try {
+			const interfaces = JSON.parse(res.stdout);
+			const tailscaleInterfaces = interfaces.filter(iface => iface.ifname.match(/tailscale[0-9]+/));
+
+			return tailscaleInterfaces.map(iface => {
+				const parsedInfo = {
+					name: iface.ifname
+				};
+
+				const addr_info = iface.addr_info || [];
+				addr_info.forEach(addr => {
+					if (addr.family === 'inet') {
+						parsedInfo.ipv4 = addr.local;
+					} else if (addr.family === 'inet6') {
+						parsedInfo.ipv6 = addr.local;
+					}
+				});
+
+				parsedInfo.mtu = iface.mtu;
+				parsedInfo.rxBytes = '%1024mB'.format(iface.stats64.rx.bytes);
+				parsedInfo.txBytes = '%1024mB'.format(iface.stats64.tx.bytes);
+
+				return parsedInfo;
+			});
+		} catch (e) {
+			ui.addNotification(null, E('p', {}, _('Error parsing interface info: %s.').format(e.message)));
+			return [];
+		}
+	},
+
+	pollData(container) {
+		poll.add(L.bind(function () {
+			return this.load().then(L.bind(function (data) {
+				dom.content(container, this.renderContent(data));
+			}, this));
+		}, this));
```

Comment:
You need to use a lambda here to reach `this` without the nested binding.
```suggestion
		poll.add(async () => {
			const data = await this.load();
			dom.content(container, this.renderContent(data));
		});
```
This omits the error handling, so it's up to you figure out if it's needed, e.g. what if `fs.exec` fails?